### DISCUSSION TITLE
Engine page cursor for power management

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#566] [Completed things] [answered]
From Satiel86: On the engine page is it possible to put a cursor (like zibo) that tell you how much throttle you want to apply? For people without realistic hardware throttle can be very useful

**Comment by aeroniemi** [comment]
will be added as part of the -8 dataset rebuild

**Comment by aeroniemi** [comment, marked as answer]
or apparently now as the -8 demo data has now made it into cygnus

**Reply by Simleybob** [reply]
Fantastic, I will test it!
